website/views/shared/tasks/task_view/index.jade

```diff
@@ -4,23 +4,39 @@
   // Habits
   .task-actions(ng-if='::task.type=="habit"')
     // score() is overridden in challengesCtrl to do nothing
+    input.visuallyhidden(
+      class="habit",
+      ng-if='task.up',
+      type='checkbox',
+      ui-keypress='{13:"score(task, up)"}')
     a(ng-if='task.up', ng-click='applyingAction || score(task,"up")')
       span.glyphicon.glyphicon-plus
+    input.visuallyhidden(
+      class="habit",
+      ng-if='task.down',
+      type='checkbox',
+      ng-init='direction = "down"'
+      ui-keypress='{13:"score(task, direction)"}' )
     a(ng-if='task.down', ng-click='applyingAction || score(task,"down")')
       span.glyphicon.glyphicon-minus
 
   // Rewards
   span(ng-if='::task.type=="reward"')
+    input.visuallyhidden(
+      class="reward",
+      type='checkbox',
+      ui-keypress='{ 13:"score(task, down)" }')
```

Comment:
This isn't actually working, because `down` is undefined. If you sync, you'll gain back the gold that was spent. Same thing happens with the habits. It should be: `ui-keypress='{ 13:"score(task, \'down\')" }'`.